docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -23,44 +23,58 @@ Scope of Modeling
 Concept Model Diagram
 ---------------------
 
-The following four diagrams are placeholders; they will be combined into a
-single model encompassing all three interventions.
+.. image:: lsff_concept_model_diagram.svg
 
-Vitamin A Fortification
-+++++++++++++++++++++++
+Model Components
+----------------
 
-.. image:: vitamin_a_diagram.svg
+Demographics
+++++++++++++
 
-Iron Fortification
-++++++++++++++++++
+GBD Causes
+++++++++++
 
-.. image:: iron_neonatal_diagram.svg
+:ref:`Measles <2017_cause_measles>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: iron_detail_diagram.svg
+:ref:`Diarrheal Diseases <2017_cause_diarrhea>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Folic Acid Fortification
-++++++++++++++++++++++++
+:ref:`Neural Tube Defects <2017_cause_neural_tube_defects>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: folic_acid_diagram.svg
+Lower Respiratory Infections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
+*Neonatal encephalopathy* due to birth asphyxia and birth trauma
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   Add Coverage Gap layer. Add specific neonatal causes. Combine diagrams.
+*Neonatal sepsis* and other neonatal infections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Model Components
-----------------
+Haemolytic disease and other *neonatal jaundice*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Demographics
-++++++++++++
+*Other neonatal* disorders
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PAF-of-1 Cause/Risk Pairs
++++++++++++++++++++++++++
+
+Vitamin A Deficiency / Vitamin A Deficiency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-GBD Risks and Causes
+Dietary Iron Deficiency / Iron Deficiency
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Additional GBD Risks
 ++++++++++++++++++++
 
-:ref:`Measles <2017_cause_measles>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Low Birth Weight and Short Gestation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-:ref:`Diarrheal Diseases <2017_cause_diarrhea>`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Risk-Cause Relationships
```

Comment:
Let's call these risk-outcome relationships.  Things we know exist but have not modeled yet: risk-risk relationships where exposure to one risk increases your exposure to another.

Reply:
Ok, I'll change this.